native/.circleci

```diff
@@ -0,0 +1 @@
+../.circleci
```

Comment:
Does this work on windows?

Reply:
:shrug: 
Probably not :D

Reply:
Does not work on windows...

Reply:
On windows there should also be symlinks, but they seem to work slightly different. Tried using a basic link, but IntelliJ doesn't know what to do with that. I only found that you could use content roots in intellij, but it doesn't really look that fancy, so I would not recommend them.